### PR TITLE
Git order

### PR DESCRIPTION
**Pull request by fasterius**
This branch contains work done to re-order to the Git tutorial so that it is clearer what is git (the version control system) and what is GitHub (working remotely with git version control). It updates and clarifies the whole tutorial, including new images that are identical to what the students should be seeing if they follow the steps and their new order. It also shortens the table of contents for the git session to a more easily viewed state.

### CODE REVIEW

**Comment by fasterius** [comment]
Thanks! I've added some changes to your first two points. Point three could definitely be discussed and changed as we see fit.

**Comment by johnne** [comment]
I think this new order looks great! I also agree with the suggestion from @verku to add the account creation to under **Working remotely**. Nice work!

File: docs/git.md [hunk 1]
```diff
@@ -346,675 +331,763 @@ Here we used `rm Dockerfile` to delete the file and `git add Dockerfile` to
...
+    We learned how to push local branches to a remote with `git push origin
+    <branch>` and how to delete remote branches with `git push origin --delete
+    <branch>`.
 
-* To get a visual view of your branches and commits you can use the command:
+### Remote tags
+
+Remote tags work, yet again, very similarly to how they work locally, but they
+are not included when you do a normal push. To push tags to GitHub you need to
+supply the `--tags` flag:
```
Comment (johnne) [inline docs/git.md]:
```suggestion
### Sharing tags

Your local repository tags are not included when you do a normal push. To push tags to the remote you need to
supply the `--tags` flag to the `git push` command:
```

**Comment by fasterius** [comment]
Great! I've added the requested changes, moved the creation of the GitHub account text and merged.

**Pull request merged**